logstash-core/src/main/java/org/logstash/instrument/witness/PluginsWitness.java

```diff
@@ -20,10 +30,12 @@ public PluginsWitness() {
         this.inputs = new ConcurrentHashMap<>();
         this.outputs = new ConcurrentHashMap<>();
         this.filters = new ConcurrentHashMap<>();
-     }
+        this.codecs = new ConcurrentHashMap<>();
```

Comment:
@original-brownbear - in order to tweak concurrency level, I also have to tweak the initial capacity and load factor.  Also, the javadoc states the default of 16 is "Unused but defined for compatibility with previous versions of this class".  

I am going to leave this as default, since this feels alot like pre-mature optimization.